openmls_test/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "openmls_test"
-version = "0.1.0"
```

Comment:
This is the first release of this crate. Let's stay with `0.1.0-pre.1`

Reply:
done